Fix singularity hub link

serious-steve opened the pull request:
Fixes https://github.com/singularityware/singularityware.github.io/issues/42

serious-steve committed:
update info about singularity hub, remove dead link

serious-steve committed:
update info about singularity hub, remove dead link

serious-steve committed:
commenting out dead link for now

serious-steve committed:
commenting out dead link for now

vsoch commented:
Thanks! Could you rebase with master to add recent changes (eg, the circle yml file) so that testing is triggered and we can preview your changes?

serious-steve committed:
reapply fix
Merge branch 'FixSingularityHubLink' of https://github.com/c1t4r/singularityware.github.io into FixSingularityHubLink

serious-steve commented:
I did what you suggested, and either I made an mistake or I don't understand what happened.
There are five commits instead of two, and the PR looks identical to me.
I ran git rebase upstream/master on my branch, and committed. Is that what you aked me for?

Sorry... new to gitlabs development practises ... :)

vsoch commented:
I think you did it correctly (maybe with a little extra, haha) and it's strange that the CI didn't trigger - it may be something about the initial push not having circle, or that circle is down. For now I think the changes look good so I'll merge without preview. Thanks for your help with this!
